concent_api/verifier/tasks.py

```diff
@@ -31,20 +32,22 @@
 from .constants import VERIFICATION_RESULT_SUBTASK_STATE_UNEXPECTED_LOG_MESSAGE
 from .utils import clean_directory
 from .utils import prepare_storage_request_headers
+from .utils import run_blender
 from .utils import store_file_from_response_in_chunks
+from .utils import unpack_archive
 
 
 logger = logging.getLogger(__name__)
 
 
 @shared_task
 def blender_verification_order(
-    subtask_id:             str,
-    source_package_path:    str,
-    result_package_path:    str,
-    output_format:          Enum,  # pylint: disable=unused-argument
-    scene_file:             str,  # pylint: disable=unused-argument
-    report_computed_task:   message.ReportComputedTask,
+    subtask_id: str,
+    source_package_path: str,
+    result_package_path: str,
+    output_format: Enum,  # pylint: disable=unused-argument
```

Comment:
isn't it `BlenderSubtaskDefinition.OutputFormat`?

Reply:
:+1: